docs/tutorials/shop-page.md

```diff
@@ -0,0 +1,396 @@
+# Shop page
+_Goal: Create a page to sell developer products in a game._
+
+[add an image of the result we are expecting]
+
+## Top Component
+Let's start by creating the top level component, that will be the page containing the items for sale in the game. When creating a new component, we need to think about what will this component manage itself and what will be given to that component from the parent component.
+
+In our case, the ShopPage component we're making will receive it's products from a parent component: that means that the product items will be received as props in our component.
+
+> Props or state?
+>
+> This is a question that often comes to our mind when we start working Roact. Remember that state is internal to the component. Things that are relevant only to a component should go in the state.
+>
+> On the other hand, props are used to pass data from parent components to child components. It implies that both parent component and it's child are aware of the structure of the props dictionary and depends on it.
+
+For our first draft of the component, we will define some props to be able to pass parameters to customize our shop page. We will start by adding the following:
+ - **frameProps:** a dictionary of props that will be passed to the scrolling frame to change it's visuals
```

Comment:
Could frameProps be removed? It seems strange to include as changing the style of the ScrollingFrame isn't part of what we want to be teaching. It seemed like you were modifying frameProps to change the CanvasSize but could that be handled differently without bloating the tutorial?

Reply:
My goal was to make the frame style accessible from the top level component, so someone could easily modify it from the script used to mount the component to fit his style

Having it also allow to tell the reader that the props cannot be mutated in the ShopPage component.

How would you change it?